ambry-frontend/src/main/java/com/github/ambry/frontend/UserQuotaRequestCostPolicy.java

```diff
@@ -90,14 +91,8 @@ private double calculateCapacityUnitCost(RestRequest restRequest, RestResponseCh
     RequestPath requestPath = getRequestPath(restRequest);
     switch (restRequest.getRestMethod()) {
       case POST:
-        long contentSize = 0;
```

Comment:
There is no need to remove this logic since the request cost is also used in C2S events.

Reply:
fixed.